marketing_blocks/managers.py

```diff
@@ -0,0 +1,18 @@
+from django.db import models
+
+
+class MarketingBlocksManager(models.Manager):
+
+    def get_active_blocks_by_position(self, position):
+        """Returns all the active blocks for a specific position"""
+
+        return self.filter(position=position, active=True)
```

Comment:
Actually I can have two active blocks for one position when calling this method.

Here is what we have in the `models.py`:
```python
qs = MarketingBlock.objects.get_active_blocks_for_position(self.position)
if self.pk:
    qs = qs.exclude(pk=self.pk)
```

Maybe we should handle this differently but we can't use a `get` instead of a `filter` as it is.

Reply:
Fixed by 2709b6f